techniques/applications/systemUpdate/1.0/systemUpdate.st

```diff
@@ -76,16 +80,18 @@ bundle agent rudder_system_update {
                                                                             "1", # periodicity days
                                                                             "stateless"
                                                                           );
+      "enable_reporting"   usebundle => enable_reporting;
```

Comment:
Not needed, it is enabled by the metadata.xml

Reply:
it is needed because i disabled it jut above to call a schedule_simple

Reply:
and still have the logs of the schedule_simple